Source/Bugsnag.m

```diff
@@ -161,7 +177,7 @@ + (void)clearTabWithName:(NSString *)tabName {
 }
 
 + (BOOL)bugsnagStarted {
-    if (self.notifier == nil) {
+    if (!self.notifier.started) {
```

Comment:
Does this handle the notifier being `nil`?

Reply:
Yes, as nil will be coerced to false.